migrations/update_legacy_filtered_positives.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import logging.config
+from datetime import datetime
+from crawler.helpers.general_helpers import get_config
+from migrations.helpers.update_filtered_positives_helper import (
+    update_filtered_positive_fields,
+    update_mlwh_filtered_positive_fields,
+    update_mongo_filtered_positive_fields,
+)
+from migrations.helpers.update_legacy_filtered_positives_helper import (
+    legacy_mongo_samples,
+    get_cherrypicked_samples_by_date,
+    v0_version_set,
+    split_mongo_samples_by_version,
+)
+from crawler.constants import (
+    V0_V1_CUTOFF_TIMESTAMP,
+    V1_V2_CUTOFF_TIMESTAMP,
+)
+from crawler.filtered_positive_identifier import (
+    FILTERED_POSITIVE_VERSION_0,
+    FILTERED_POSITIVE_VERSION_1,
+    FILTERED_POSITIVE_VERSION_2,
+)
+from migrations.helpers.dart_samples_update_helper import extract_required_cp_info
```

Comment:
I move this method in my open PR to `shared_helper.py`. Just be aware that one of us may have to change this depending who's branch gets merged in first

Reply:
OK 👍